pywbem_mock/_wbemconnection_mock.py

```diff
@@ -1068,124 +1300,91 @@ def _display_objects(obj_type, object_repo, namespace, cmt_begin, cmt_end,
+        implementation provided with this method as the `provider` parameter is
+        to be executed as the request response method for the namespaces
+        defined in the `namespaces` parameter, the provider type defined in the
+        provider 'provider_type` attribute of the `provider` and the classes
+        defined in the provider `provider_classnames` attribute of the
+        `provider`.
 
-        Providers can only be registered for request response methods
-        CreateInstance and DeleteInstance and InvokeMethod.
+        The provider registration process includes:
 
-        Multiple providers may be registered for the same classname in
-        different namespaces
+        1. Validation that the namespaces defined for the provider exist.
+        2. Validation that the superclass of the provider is consistent with
+           the `provider_type` attribute defined in the provider.
+        3. Installation of any CIM classes defined by the provider
+           (`provider_classnames` attribute) including installation of
+           dependencies for these classes using the `schema_pragma_files` to
+           locate the search directories for dependencies.
+        4. Adding the provider to the registry of user_providers so that any
+           of the request methods defined for the `provider_type` are
+           passed to this provider in place of the default request processors.
 
-        Parameters:
+        Providers can only be registered for the following request response
+        methods:
 
-          namespaces (:term:`string` or :class:`py:list` of :term:`string`):
-            Namespace or namespaces for which the provider is being registered.
-            If no namespaces are defined, the connection default namespace
-            is used.
+        1. provider_type = 'instance': CreateInstance, ModifyInstance, and
+           DeleteInstance
 
-          classnames (:term:`string` or :class:`py:list` of :term:`string`):
-            Classname or classnames for which the provider is being
-            registered. At least one classname is required.
+        2. provider_type = 'method': InvokeMethod.
 
-          provider_type (:term:`string`):
-            Keyword defining the type of request the provider will service.
-            The allowed types are instance (keyword 'instance') which responds
-            to the request responder methods defined in InstanceWriteProvider
-            (ex. CreateInstance) and method (keyword 'method') which responds
-            to the InvokeMethod.
-
-          provider (subclass of :class::class:`pywbem_mock:InstanceWriteProvider`):
-            The user provider class which is a subclass of
+        Each classname in a particular namespace may have at most one provider
+        registered
+
+        Parameters:
+
+          provider (instance of subclass of :class:`pywbem_mock:InstanceWriteProvider`):
```

Comment:
Change colon between pywbem_mock and class name to a dot.
Occurs two times in this parameter description.

Reply:
DONE